styles/components/contact.css

```
@@ -0,0 +1,23 @@
+.contact {
+  padding: 40px 40px 20px;
+  text-decoration: none;
+  color: whitesmoke;
+}
+
+.mail {
+  text-decoration: none;
+  color: whitesmoke;
+}
+.telephone {
+  text-decoration: none;
+  color: whitesmoke;
+}
+
+.socials {
+  text-decoration: none;
+  color: whitesmoke;
+}
+
+.projects {
+  padding: 50px;
+}
```

I would stick to BEM convention and name the elements after the block class:
```suggestion
.contact__mail {
  text-decoration: none;
  color: whitesmoke;
}
.contact__telephone {
  text-decoration: none;
  color: whitesmoke;
}

.contact__socials {
  text-decoration: none;
  color: whitesmoke;
}

.contact__projects {
  padding: 50px;
}
```